setup.cfg

```diff
@@ -36,7 +36,7 @@ install_requires =
     aiohttp-jinja2==1.5
     aiohttp-session[secure]==2.10.0
     aiohttp==3.8.1
-    aioipfs@git+https://github.com/aleph-im/aioipfs.git@hoh-more-exceptions
+    aioipfs@git+https://github.com/odesenfans/aioipfs.git@od-upgrade-pubsub-format-for-0.11.0
```

Comment:
Can this branch be hosted on the `aleph.im` orga ?

Reply:
Yes, you just need to approve/merge https://github.com/aleph-im/aioipfs/pull/2.